tests/test_util.py

```diff
@@ -255,8 +256,21 @@ def test_B9_persist_temp_file(self):
     dest_path = os.path.join(dest_temp_dir, self.random_string())
     tmpfile = tempfile.TemporaryFile()
     tmpfile.write(self.random_string().encode('utf-8'))
-    securesystemslib.util.persist_temp_file(tmpfile, dest_path)
+
+    # Write a file with custom permissions
+    # stat.S_IRUSR: user has read permissions
+    # stat.S_IWUSR: user has write permissions
+    mode= stat.S_IRUSR | stat.S_IWUSR
```

Comment:
```suggestion
    mode = stat.S_IRUSR | stat.S_IWUSR
```

Reply:
Fixed in [6250c79](https://github.com/secure-systems-lab/securesystemslib/pull/231/commits/6250c790c9b9c317e661b345b345bede42c77cb9)